### PR TITLE
Adding a new rule for detecting this variant of secneo that includes a new shared library

### PR DESCRIPTION
**Pull request by P0r0**
Related issue >> https://github.com/rednaga/APKiD/issues/118

### CODE REVIEW

File: apkid/rules/apk/packers.yara [hunk 1]
```diff
@@ -88,6 +88,21 @@ rule secneo : packer
...
 
+rule secneo_a : packer
+{
+  meta:
+    description = "SecNeo"
+    url = "http://www.secneo.com"
+
+  strings:
+    $lib1 = "libdexjni.so"
+    $lib2 = "libdexjni%s.so"
```
Comment (enovella) [inline apkid/rules/apk/packers.yara]: Shouldn't it be a regex?    `$lib2 = /libdexjni%s.so/`

File: apkid/rules/apk/packers.yara [hunk 2]
```diff
@@ -88,6 +88,21 @@ rule secneo : packer
     is_apk and any of ($encrypted_dex, $encryptlib2, $encryptlib1)
 }
 
+rule secneo_a : packer
+{
+  meta:
+    description = "SecNeo"
```
Comment (enovella) [inline apkid/rules/apk/packers.yara]: Any fancy name to distinguise between versions?  //cc @strazzere

File: apkid/rules/apk/packers.yara [hunk 3]
```diff
@@ -88,6 +88,21 @@ rule secneo : packer
     is_apk and any of ($encrypted_dex, $encryptlib2, $encryptlib1)
 }
 
+rule secneo_a : packer
```
Comment (enovella) [inline apkid/rules/apk/packers.yara]: Good naming!

**Comment by enovella** [comment]
Was this library having this protector too? https://github.com/chago/ADVMP

**Comment by enovella** [comment]
@CalebFenton anyone reviewing that?

**Comment by CalebFenton** [comment]
Any chance of getting the sample or sample hash here?

**Comment by CalebFenton** [comment]
```$ shasum -a 256 *
f5d7985e2add50fce74c99511512084845558ac996ce66f45e633c9495d78400  cn.jsb.china.apk
4f2d135c1efe3d68849887c6a879784f999b8cab82546aca1960959fa82dc357  com.rytong.app.bankhx.apk
8b88daeb20d7895733b24ad67cd71a4a87a65a26bfa605cfc03a3fbc6ced40c1  com.unionpay_6.0.3_124.apk```

```

**Pull request merged**